main.py

```diff
@@ -654,6 +653,35 @@ async def on_message(message):
                 except Exception:
                     pass
 
+    if text.lower() == "tac":
+        channeled = Channel.get_or_none(channel_id=message.channel.id)  # I'm just gonna copy the relevant stuff over from "cat" but have unique variable names
+        if channeled and channeled.cat:
+            try:
+                goodmessage = await message.channel.fetch_message(channel.cat)
+            except Exception:
+                await message.channel.send(f"oopsie poopsie i cant access the original message so i can't be sure if {message.author.mention} got the reverse ach or not so imma play it safe and say no achievement received")
+                channeled.cat = 0
+                channeled.save()
+                return
+            goodcontent = goodmessage.content
+            party_type = None
+            
+            for v in allowedemojis:
+                if v in goodcontent:
+                    party_type = v
+                    break
+
+            if not party:
+                return
+
+            for i in type_dict.keys():
+                if i.lower() in party_type:
+                    thecattype = i
+                    break
+
+            if thecattype == "Reverse":
```

Comment:
all of this can be replaced with `if "reversecat" in goodmessage.content:`

Reply:
I was thinking /changemessage would mess up the cat spawn but then I noticed there's an actual check to prevent that so good job